### DISCUSSION TITLE
Slow scrolling while in a scrollable container

### DISCUSSION THREAD

**Discussion by ericchernuka** [clauderic/dnd-kit#796] [Q&A]
I'm wondering if anyone has any ideas or direction to investigate as to why scrolling within this modal is incredibly slow or won't scroll at all while dragging. Any thoughts are appreciated. Thanks!

https://user-images.githubusercontent.com/243155/173620373-abb438c4-44d0-4918-a514-c6d75c82bc9b.mp4

**Comment by clauderic** [comment]
Please provide a minimal reproduction case (ideally in CodeSandbox) that demonstrates the issue so I can investigate.

**Reply by ericchernuka** [reply]
You bet. Sorry @clauderic!
